src/main/resources/validationrules/prm/prm-role-mapping-validation.drl

```diff
@@ -0,0 +1,39 @@
+package validationrules.core;
+import uk.gov.hmcts.reform.roleassignment.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.roleassignment.domain.model.Request;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.Status;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RequestType;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleCategory;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleType;
+import function uk.gov.hmcts.reform.roleassignment.domain.service.common.ValidationModelService.logMsg;
+
+rule "prm_create_org_role"
+when
+    $rq: Request(
```

Comment:
Optional: you can use flag if you wish to have a switch on process.  However as `roleType == RoleType.ORGANISATION` + `roleCategory == RoleCategory.PROFESSIONAL` is unique for the new PRM process this rule should be ok to go in without a flag provided it has been through routine QA.

Reply:
decided to add flag as may be useful to disable on prod until testing complete. see commit [f8269a9](https://github.com/hmcts/am-role-assignment-service/pull/2062/commits/f8269a94896e32c2d336b6c76aa7ff585d45c1b5)